2023-01-10 10:15:58.716 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:17:56.392 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:17:58.098 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:18:24.509 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:19:26.511 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:19:28.018 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:20:57.492 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:20:59.155 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:26:46.252 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:26:47.718 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:30:53.242 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:30:54.742 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:35:29.458 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:36:47.391 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:41:15.409 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:41:16.576 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:42:04.429 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:42:05.511 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:42:32.126 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:42:55.566 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:43:19.654 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:43:49.422 ERROR response_module.py response_module - response_exception:{'message': 'list indices must be integers or slices, not str'}
2023-01-10 10:46:13.710 ERROR response_module.py response_module - response_exception:{'message': "django_parameterized_query_update() missing 1 required positional argument: 'update_tuple'"}
2023-01-10 10:47:20.274 ERROR response_module.py response_module - response_exception:{'message': 'column "i_d" does not exist\nLINE 6:             i_d = \'1\';\n                    ^\nHINT:  Perhaps you meant to reference the column "dashboard_customer.id".\n'}
2023-01-10 10:48:34.961 ERROR response_module.py response_module - response_exception:{'message': "'NoneType' object is not iterable"}
2023-01-10 10:51:53.544 ERROR response_module.py response_module - response_exception:{'message': 'column "customer_id" does not exist\nLINE 6:             customer_id = \'1\';\n                    ^\n'}
2023-01-10 10:53:16.530 ERROR response_module.py response_module - response_exception:{'message': 'Expecting value: line 1 column 1 (char 0)'}
2023-01-10 10:53:23.241 ERROR response_module.py response_module - response_exception:{'message': "'NoneType' object is not iterable"}
2023-01-10 10:54:48.738 ERROR response_module.py response_module - response_exception:{'message': 'update or delete on table "dashboard_customer" violates foreign key constraint "dashboard_vehicle_customer_id_c64ba67f_fk_dashboard_customer_id" on table "dashboard_vehicle"\nDETAIL:  Key (id)=(1) is still referenced from table "dashboard_vehicle".\n'}
2023-01-10 10:58:53.185 ERROR response_module.py response_module - response_exception:{'message': "'NoneType' object is not iterable"}
2023-01-10 10:59:12.351 ERROR response_module.py response_module - response_exception:{'message': 'syntax error at or near "UPDATE"\nLINE 2:             UPDATE public.dashboard_customer\n                    ^\n'}
2023-01-10 11:01:39.629 ERROR response_module.py response_module - response_exception:{'message': "'NoneType' object is not iterable"}
2023-01-10 11:02:00.334 ERROR response_module.py response_module - response_exception:{'message': 'update or delete on table "dashboard_customer" violates foreign key constraint "dashboard_vehicle_customer_id_c64ba67f_fk_dashboard_customer_id" on table "dashboard_vehicle"\nDETAIL:  Key (id)=(1) is still referenced from table "dashboard_vehicle".\n'}
2023-01-10 11:03:43.546 ERROR response_module.py response_module - response_exception:{'message': 'update or delete on table "dashboard_customer" violates foreign key constraint "dashboard_vehicle_customer_id_c64ba67f_fk_dashboard_customer_id" on table "dashboard_vehicle"\nDETAIL:  Key (id)=(1) is still referenced from table "dashboard_vehicle".\n'}
